modules/appeals_api/spec/factories/higher_level_reviews.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require AppealsApi::Engine.root.join('spec', 'support', 'fixture_helpers.rb')
+
```

Comment:
I am not sure why, but with this PR, I started seeing test failures [like these](https://github.com/department-of-veterans-affairs/vets-api/actions/runs/5293533052/jobs/9581966037). The error is from changes in #13001, which was merged yesterday, but I haven't seen this error locally or in other PRs until now. I'm hoping the second commit I pushed here will fix this for good.

Reply:
I've been hitting unit test fails that I resolved with a require also